setup.py

```diff
@@ -21,15 +24,22 @@ def load_about():
     return about
 
 
-def load_requirements():
+def load_requirements() -> List[str]:
     with io.open(
         os.path.join(HERE, "requirements", "base.in"), "rt", encoding="utf-8"
     ) as f:
         return [line.strip() for line in f if is_requirement(line)]
 
 
+def load_dev_requirements() -> List[str]:
+    with io.open(
+        os.path.join(HERE, "requirements", "dev.in"), "rt", encoding="utf-8"
+    ) as f:
+        return [line.strip() for line in f if is_requirement(line)]
+
+
 def is_requirement(line):
-    return not (line.strip() == "" or line.startswith("#"))
+    return not (line.strip() == "" or line.startswith("#") or line.startswith("-"))
```

Comment:
I don't understand: which case is this check supposed to cover?

Reply:
Hi @regisb , when running the unit tests, the setup will load the requirements for unit tests (i.e. `load_dev_requirements()`) via the [`tests_require`](https://github.com/overhangio/tutor/blob/3e61feced4b6b126bddd631e29ba5782fe4a6489/setup.py#L81). This will then cause error when trying to load the requirements because the `dev.in` file contains this [specific line `-r base.txt`](https://github.com/overhangio/tutor/blob/3e61feced4b6b126bddd631e29ba5782fe4a6489/requirements/dev.in#L1), which needs to be ignored by the `tests_require` like the other comments.

Reply:
I have reverted the changes here back to the original usage without the new load_dev_requirements() function via my repo PR  [#8](https://github.com/takato1314/tutor/pull/8).